### ISSUE TITLE
Nothing happening after install

### ISSUE THREAD

**Issue by chrismatheson** [w0lfschild/Flashlight#70]
I’ve had a quick look through the issues but can’t seem to see anyone with a similar problem.

I downloaded and ran v1.2 and the app itself comes up fine, however none of the triggers do anything in spotlight.

Have I maybe missed an installation step?

Thanks in advance for any help!

**Comment by kasperii** [comment]
_I wonder the same! Downloaded and all, but nothing is different in spotlight._

Edit: never mind, it is working now

**Comment by w0lfschild** [comment]
What version of macOS?

**Comment by chrismatheson** [comment]
@w0lfschild I read it, but glossed over. I think this might be a result of SIP ? I haven't disabled mine, as much as a REALY want to extend Spotlight, I think disabling SIP globally (instead of for this single application) seems too risky for me.

Edit: sorry I dint answer your question 10.14.6

**Comment by AlbertoMontalesi** [comment]
Same here, installed but nothing comes up on Spotlight. Macos 10.15.4

**Comment by ZhenningLang** [comment]
Similar problem, macOS Catalina, ver 10.15.4

**Comment by rikhoffbauer** [comment]
experiencing the same on macOS 10.14.6 with SIP disabled

@w0lfschild any updates on this?

**Comment by w0lfschild** [comment]
Have you installed MacForge?

**Comment by rikhoffbauer** [comment]
@w0lfschild yes I have installed the latest version of MacForge

**Comment by lachlancresswell** [comment]
I'm experiencing the same issue. MacForge is installed. Flashlight is installed. I can open and download commands. Opening Spotlight and typing "emoji grin" returns nothing except regular Spotlight results. I've disabled SIP and Library Validation, though, there isn't any information in the readme on whether or not this is required. This is on 10.15.7.

EDIT:
Managed to get it working by reinstalling flashlight. Is it possible that Flashlight needs to be installed after disabling Library Validation?

EDIT 2:
Spotlight also now shows in the "Mange Plugins" tab in MacForge whereas it didn't earlier, so, I'm assuming the install did happen correctly initially.
